src/dashboard/src/components/advanced_search.py

```diff
@@ -59,7 +71,9 @@ def search_parameter_prep(request):
             except:
                 types.insert(index, '')
 
-            index = index + 1
+        for index, field in enumerate(fields):
```

Comment:
Ahh, okay.  Can you add that as a comment?  Otherwise it looks like a subtle overwrite bug.